app/models.py

```diff
@@ -0,0 +1,58 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class CustomUser(User):
+    ADMIN = 'admin'
+    MANAGER = 'manager'
+    REGULAR = 'regular'
+
+    ROLE_CHOICES = (
+        (ADMIN, 'Admin'),
+        (MANAGER, 'Manager'),
+        (REGULAR, 'Regular User'),
+    )
+
+    role = models.CharField(
+        max_length=10,
+        choices=ROLE_CHOICES,
+        default=REGULAR,
+    )
+
+
+class Task(models.Model):
+    STATUS_CHOICES = (
+        ('New', 'New'),
```

Comment:
same as here: https://github.com/Numeer/BattleShip/pull/7/files#r1278852078

and make constants as above